js/jsfunfuzz/gen-grammar.js

```diff
@@ -271,7 +271,7 @@ var statementMakers = Random.weighted([
   //{ w: 3, v: function(d, b) { return "var opn = Object.getOwnPropertyNames(" + makeId(d, b) + "); for (var j = 0; j < opn.length; ++j) { addPropertyName(opn[j]); }"; } },
 ]);
 
-if (typeof oomTest == "function" && engine != ENGINE_SPIDERMONKEY_MOZILLA45) {
+if (typeof oomTest == "function" && engine != ENGINE_JAVASCRIPTCORE) {
```

Comment:
Hmmm, I combined the removal of ESR45 code and addition of jscore detection here. I guess they can be separate changes. oomTest is something specific to SpiderMonkey hence the latter change.